packages/blockchain-wallet-v4-frontend/src/components/Alerts/template.js

```diff
@@ -51,6 +52,23 @@ const Alerts = props => {
   )
 }
 
+const SuccessCheckmark = () => {
+  return (
+    <svg
+      width='24'
+      height='24'
+      viewBox='0 0 24 24'
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'
+    >
+      <path
+        d='M21.3333 0H2.66667C1.18667 0 0 1.2 0 2.66667V21.3333C0 22.8 1.18667 24 2.66667 24H21.3333C22.8133 24 24 22.8 24 21.3333V2.66667C24 1.2 22.8133 0 21.3333 0ZM9.33333 18.6667L2.66667 12L4.54667 10.12L9.33333 14.8933L19.4533 4.77333L21.3333 6.66667L9.33333 18.6667Z'
+        fill='#0C6CF2'
+      />
+    </svg>
+  )
+}
+
```

Comment:
Should we add an icon?

Reply:
yeah, let's add it as an icon just to be safe and keep things clean.  @seanblockchain let me know if you need help doing this and i can walk you through the "process" ;)